handler.go

```diff
@@ -196,8 +211,8 @@ func (fm *Frontman) Run(inputFilePath *string, outputFile *os.File, interrupt ch
 
 	outputLock := sync.Mutex{}
 
-	var results []Result
 	for true {
```

Comment:
To be honest I don't really understand this loop.
It's doing too many things and and the nesting of `if` and `for` doesn't make it easier.
So I can't really comment on the changes done here

Reply:
I also don't like it. I will try to split it in the next PR

Reply:
Thanks